test/data/dir-pcap/labels.txt

```diff
@@ -0,0 +1,203 @@
+Item,Label
```

Comment:
I added (needless) hierarchy to the pcap test directory to make it easier to test recursion.